.travis.yml

```diff
@@ -2,9 +2,8 @@ sudo: false
 language: go
 
 go:
- - 1.7.x
- - 1.8.x
- - 1.9.x
+  - 1.9.x
+  - 1.10.x
 
 script:
- - go test -short ./...
+  - go test ./...
```

Comment:
I was working with @TheTincho on packaging updates, and we noticed one test that was failing there, and I found it reproduced locally, but not in the short test.  I figured it would be safer to run the full test.

Reply:
Certainly it's safer, but if the CI times out, it's of no use.

Please put the `-short` back and run full tests manually when required.

Reply:
The CI doesn't seem to timeout anymore.

Reply:
Hmm, perhaps Travis uses faster hardware by now. A Travis job times out after 10m without output. That's probably the reason why we timed out sometimes. Running all test takes about 6m now on Travis. I guess with slightly slower hardware, we could still trigger the timeout now and then. In different news, I find a responsive CI quite important for development and the discipline to keep the CI green. Removing `-short` more than doubles the Travis runtime (from ~2m30s to ~6m). That's getting into annoying zone. There are only two tests that get activated by removing the `-short`, and they are rather specific concurrency tests (which ideally should also be run with the race detector, which we are not doing on the CI anyway).

My take is that the per-PR-CI tests are not supposed to replace thorough tests you would apply prior to release but are supposed to give you fast feedback on most of the tests during development.